data/validate_metadata_docu_files.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs')
+
+const meta = require('./metadata.json')
+
+let allFilesExist = true
+
+const usageDocKeys = [
+  "generic",
+  "specific",
+  "csaf_base",
+  "csaf_security_incident_response",
+  "csaf_informational_advisory",
+  "csaf_security_advisory",
+  "csaf_vex"
+]
```

Comment:
Why not extract them from the schema itself? Then, they would be up-to-date whatever changes... Not tested, but something like the following could work:
```suggestion
const usageDocKeys = meta.properties.metadata.items.properties.user_documentation.properties

Reply:
That's a really nice idea. It requires additionally reading the schema, but I think that's totally worth the reduced maintenance needed. I added it.